src/markdown_generators.js

```diff
@@ -155,14 +156,16 @@ const generateCourseSectionFrontMatter = (
   courseId,
   isMediaGallery,
   videoUids,
-  type
+  type,
+  description
 ) => {
   /**
     Generate the front matter metadata for a course section
     */
   const courseSectionFrontMatter = {
-    uid:   helpers.addDashesToUid(pageId),
-    title: helpers.replaceIrregularWhitespace(title)
+    uid:         helpers.addDashesToUid(pageId),
+    title:       helpers.replaceIrregularWhitespace(title),
+    description: description
```

Comment:
Do we need to worry about any potential "irregular whitespace" here like in the title?

Reply:
Good call.  I ran this through all courses both with and without and there were a lot of descriptions that ended up with literal line breaks (`\r\n`) written into them.  Passing it through `replaceIrregularWhitespace` fixed that.